woocommerce/woocommerce-android
# Enable variable product creation

Summary
==========
Fixes issue #4010 by enabling the Variable Product creation from the Product List FAB. 

Since a Variable Product needs to also be existent at the site in order to make available any Attributes or Variations creating/editing, I decided to submit the new Product to the site immediately after the creation request but flagged with the Draft status, this way all Variable Product modifications will be available, and the product will be published only after the merchant think it's done with the editing.

I'll deliver in an upcoming PR a bigger change that allows us to fully manage Attributes and Variations without having to post the product beforehand, so we can at least enable the Variable Product creation right away.

How to Test
==========
| Go to the Product List View  | Click at the FAB and verify the `Variable Product` option and click on it | Verify that a new Variable Product is created and it's possible to edit everything on it |
| ------------- | ------------- | ------------- |
| ![Screenshot_20210516_211320](https://user-images.githubusercontent.com/5920403/118417715-c1056e00-b68b-11eb-9ed2-fe1c057430e4.png) | ![Screenshot_20210516_211323](https://user-images.githubusercontent.com/5920403/118417722-c4005e80-b68b-11eb-8676-ae39d7a6b8ee.png) | ![Screenshot_20210516_211341](https://user-images.githubusercontent.com/5920403/118417726-c662b880-b68b-11eb-8c56-fd048181da19.png) |


Update release notes:
